packages/ra-ui-materialui/src/detail/showFieldTypes.js

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import Datagrid from '../list/Datagrid';
```

Comment:
Why is this file still in JS ?

Reply:
it's in `detail` - this part isn't migrated to Typecript yet (it's the last part to migrate)